update.go

```diff
@@ -151,8 +156,9 @@ other options are ignored.
 			for opt, dest := range map[string]*uint64{
 				"cpu-period":         r.CPU.Period,
 				"cpu-quota":          r.CPU.Quota,
-				"cpu-share":          r.CPU.Shares,
+				"cpu-shares":         r.CPU.Shares,
```

Comment:
alright, I think this change needs its own commit (or PR better) since you're changing a flag, aren't you?